podcast/podcast.go

```diff
@@ -223,33 +224,41 @@ func (p *Podcasts) isAudio(rawItemURL string) (bool, error) {
 	return p.tagReader.CanRead(itemURL.Path), nil
 }
 
-func itemToEpisode(podcastID, size, duration int, audio string, item *gofeed.Item, podcastTitle string) *db.PodcastEpisode {
+func getPodcastEpisodeAuthor(episodeAuthor string, podcastAuthor string) string {
+	if len(episodeAuthor) > 0 {
+		return episodeAuthor
+	}
+
+	return podcastAuthor
+}
+
+func itemToEpisode(podcast *db.Podcast, size, duration int, audio string, item *gofeed.Item) *db.PodcastEpisode {
 	return &db.PodcastEpisode{
-		PodcastID:   podcastID,
+		PodcastID:   podcast.ID,
 		Description: item.Description,
 		Title:       item.Title,
 		Length:      duration,
 		Size:        size,
 		PublishDate: item.PublishedParsed,
 		AudioURL:    audio,
 		Status:      db.PodcastEpisodeStatusSkipped,
-		Artist:      item.ITunesExt.Author,
-		Album:       podcastTitle,
+		Artist:      getPodcastEpisodeAuthor(item.ITunesExt.Author, podcast.Author),
```

Comment:
```suggestion
		Artist:      cmp.Or(item.ITunesExt.Author, podcast.Author),
```

Reply:
I was trying to find some documentation on the best way to do this, as `||` didn't work. Thanks for that.